client/src/game/Game.js

```diff
@@ -9,9 +14,34 @@ function resizeCanvas(canvas) {
   canvas.height = 667;
 }
 
+const directionKeyCodes = {
+  ArrowRight: DIRECTIONS.RIGHT,
+  ArrowLeft: DIRECTIONS.LEFT,
+  Space: DIRECTIONS.JUMPING,
+};
+
 function handleKeyDown(event) {
-  movePlayer(event.code, player);
-  rotatePlayer(event.code, player, DIRECTIONS);
+  keyState[event.code] = true;
```

Comment:
If this indicates which keys are pressed, I would rename it:
```suggestion
  activeKeys[event.code] = true;
```
or 
```suggestion
  keysState[event.code] = true;
```
or
```suggestion
  keyStates[event.code] = true;
```